### PR TITLE
Student Interfaces

### PR DESCRIPTION
**Pull request by jf-ub**
For issues:
#16 
#18 

Additions
Prototype student dashboard ui
Prototype student courses ui

Changes:
Student sidebar link to dashboard
Student enrollment ui text

### CODE REVIEW

File: WebApp/WebApp/Features/Student/Dashboard/Dashboard.cshtml [hunk 1]
```diff
@@ -1,16 +1,109 @@
-﻿<div class="page-content">
+﻿@model WebApp.Features.Student.Dashboard.DashboardController
```
Comment (vanpyrzericj) [inline WebApp/WebApp/Features/Student/Dashboard/Dashboard.cshtml]: So, you actually don't have to pass the Controller object as a model back to the cshtml file. (I'm surprised it actually let you do this, but there aren't any errors being thrown). If a model isn't needed on the page, you can just not include the @model line.

**Comment by jf-ub** [comment]
Updated based on review. Removed line passing in model to Dashboard.cshtml

File: WebApp/WebApp/Features/Student/Dashboard/Dashboard.cshtml (hunk 1 above)
Comment (jf-ub) [reply WebApp/WebApp/Features/Student/Dashboard/Dashboard.cshtml]: Makes sense. It has been updated

**Comment by vanpyrzericj** [comment]
Solves #16 and #18

**Pull request merged**